source/NeutronaWand/NeutronaWand.ino

```diff
@@ -3310,6 +3310,9 @@ void fireStream(CRGB c_colour) {
     break;
 
     case LEDS_29:
+      // Not yet supported.
```

Comment:
@nomakewan This was intentionally set. I was using the same logic for 5 LEDs to test the 29 LED setup. Please leave this as it was for internal usage.